.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  parser: "babel-eslint",
+  parserOptions: {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "modules": true
+    }
+  },
+  plugins: ["known-imports"],
```

Comment:
I think you have to add `prettier` to the list of plugins (combined with the `plugin:prettier/recommended` below) in order for Prettier to get turned on as a rule